### DISCUSSION TITLE
Global Account id not always a valid "guid"

### DISCUSSION THREAD

**Discussion by rkapoz-tech** [SAP/terraform-provider-btp#760] [Q&A]
Hi, 
I started working with the BTP Terraform Provider on some Global Accounts, but I discovered a strange behavior regarding the Global Account "IDs", and sometimes I cannot use the BTP Terraform Provider. For some of my Customers, the Global Account ID is NOT a "guid" but is a string starting with "CA1100...." (the difference is also visible in the cockpit URL), even it'a s feature set-b. Can anyone explain me this? Unfortunately on these Global Account, the TF provider seems does not work, when I reference subaccounts

**Comment by lechnerc77** [comment]
Hi @rkapoz-tech, the configuration of the Terraform provider is based on the **subdomain** of the global account (see [documentation](https://registry.terraform.io/providers/SAP/btp/latest/docs#required)). This is used to authenticate your user on log you onto the global account and there should be no issues. Be aware tthat the Terraform provider supports global accounts in feature set B only. 

What issues are you facing? Do you have any error messages you can provide
